src/world.rs

```diff
@@ -646,3 +660,67 @@ impl Default for World {
         }
     }
 }
+
+#[cfg(feature="serialize")]
+/// Structure used to serialize a world using a component group.
+pub struct WorldSerializer<'a, G> {
```

Comment:
Well, it ties in better with `serde`s traits and such this way rather than just have some special methods for it.